File: java/src/sodium/NodeTester.java
package sodium;

import junit.framework.TestCase;

public class NodeTester extends TestCase {
	public void testNode() {
		Node a = new Node(0);
		Node b = new Node(1);
		a.linkTo(b);
		assertTrue(a.compareTo(b) < 0);
	}
}

File: c#/Tests/c#/sodium/NodeTester.cs
using NUnit.Framework;

using Sodium;

namespace Tests.sodium
{
  [TestFixture]
  public class NodeTester
  {
    [Test]
    public void TestNode()
    {
      var a = new Node(0);
      var b = new Node(1);
      a.LinkTo(b);
      Assert.That(a, Is.LessThan(b));
    }
  }
}